### ISSUE TITLE
apa_table.apa_results()

### ISSUE THREAD

**Issue by mariusbarth** [crsh/papaja#391]
**Is your feature request related to a problem? Please describe.**
When creating a table from an `apa_results` object, I'm always frustrated that I have to type `$table`.

**Describe the solution you'd like**
It would be great if `apa_table()` would have an S3 method for `apa_results` objects that directly passes the `$table` element to `apa_table()`.

**Describe alternatives you've considered**
I tried being more diligent, but that didn't work.

**Add additional context**
A solution would look like the following
```
#' @rdname apa_table
#' @method apa_table apa_results
#' @export

apa_table.apa_results <- function(x, ...) {
  apa_table(x$table, ...)
}
```

What do you think? I think it would look especially great with pipes, for instance:

```
t.test(formula = yield ~ N, data = npk, paired = TRUE) %>%
  apa_print() %>%
  apa_table(caption = "My crucial test results.")
```

**Comment by crsh** [comment]
I think we have discussed this a couple of times and I'm all in favor of it. In particular, we could add an attribute to result tables that could be used to also automatically generate a table note, for example, with information on abbreviations, explained variance estimates, etc.

**Comment by crsh** [comment]
@mariusbarth This seems straight forward to implement. Do you want to do this before the upcoming release? I think we should also update examples and documentation accordingly (which might be more work than the required changes to the code).

**Comment by mariusbarth** [comment]
Has been implemented via 443f6f1f993a27ff3eddfcd5dc1882ba923c2756
